sdk/lib/fslib/btrfslib/btrfslib.c

```diff
@@ -1263,13 +1261,6 @@ static bool is_mounted_multi_device(HANDLE h, uint32_t sector_size) {
         bfs = malloc(bfssize);
 
         Status = NtDeviceIoControlFile(h2, NULL, NULL, NULL, &iosb, IOCTL_BTRFS_QUERY_FILESYSTEMS, NULL, 0, bfs, bfssize);
-        if (!NT_SUCCESS(Status) && Status != STATUS_BUFFER_OVERFLOW) {
```

Comment:
Is it certain this is not necessary anymore?

Reply:
according to upstream, it has been removed to fix a memory leak  
https://github.com/maharmstone/btrfs/commit/6ceb31763a285fc45cc6814b9fea35be5c84c306  
but the leak seems to have been managed in ReactOS with the `if (bfs) free(bfs);`  
but upstream hasn't been very explicit in his commit message... so I don't have info than that...